app/views/ckey2bibframes/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1>Ckey to Bibframe2 Conversion</h1>
```

Comment:
The users of this app all know what a Ckey is. All of the users of this app are SUL Metadata staff. The ckey is required and the form does validation on the ckey:
```
validates :ckey, presence: { message: ': You must enter a known ckey!' }
```